README.md

```diff
@@ -1,7 +1,7 @@
 ![image](https://user-images.githubusercontent.com/1029673/39358522-3d16a6aa-4a0e-11e8-9515-41909f36e70d.png)
 
 > ### VRTK.Unity.Core
-> A set of design patterns for rapidly building VR solutions in Unity3d
+> A core collection of design patterns for solving common problems
```

Comment:
```suggestion
> A collection of design patterns for solving common problems
```
I think the `common` here emphasizes the idea that we only want "core" things to be part of this. This way it also doesn't bring up any questions regarding what "core" is 😉